test/BackendGeb/FromCore/Base.hs

```diff
@@ -17,9 +17,9 @@ coreToGebTranslationAssertion ::
   Assertion
 coreToGebTranslationAssertion root mainFile expectedFile step = do
   step "Parse Juvix Core file"
-  input <- readFile . toFilePath $ mainFile
+  input' <- readFile . toFilePath $ mainFile
   entryPoint <- set entryPointTarget TargetGeb <$> testDefaultEntryPointIO root mainFile
-  case Core.runParserMain mainFile defaultModuleId mempty input of
+  case Core.runParserMain mainFile defaultModuleId mempty input' of
```

Comment:
name changes

Reply:
This change was moved to https://github.com/anoma/juvix/pull/2582